1-js/05-data-types/09-keys-values-entries/02-count-properties/task.md

```diff
@@ -15,7 +15,7 @@ let user = {
 alert( count(user) ); // 2
 ```
 
-Try to make the code as short as possible.
+سعی کنید تا جایی که می‌توانید کد را کوتاه بنویسید.
 
-P.S. Ignore symbolic properties, count only "regular" ones.
+پی‌نوشت: ویژگی‌های سمبلی را نادیده بگیرید، فقط ویژگی‌های «معمولی» را بشمارید.
```

Comment:
```suggestion
پی‌نوشت: ویژگی‌های نمادین را نادیده بگیرید، فقط ویژگی‌های «معمولی» را بشمارید.
```

Reply:
I translated **"Symbol"** as **"سمبل"** in [Symbol type](https://github.com/javascript-tutorial/fa.javascript.info/blob/master/1-js/04-object-basics/08-symbol/article.md) chapter and since "symbolic properties" refers to that, I think it's better to be translated as "ویژگی‌های سمبلی".
Maybe it's good to mention "symbolic properties" in parentheses.

Reply:
LGTM. Please add the English term as well.